keps/sig-network/0010-20180314-coredns-GA-proposal.md

```diff
@@ -0,0 +1,126 @@
+---
+kep-number: 10
+title: Graduate CoreDNS to GA
+authors:
+  - "@johnbelamaric"
+  - "@rajansandeep"
+owning-sig: sig-network
+participating-sigs:
+  - sig-cluster-lifecycle
+reviewers:
+  - "@bowei"
+  - "@thockin"
+approvers:
+  - "@thockin"
+editor: "@rajansandeep"
+creation-date: 2018-03-21
+last-updated: 2018-05-18
+status: provisional
+succeeded by: https://github.com/kubernetes/community/pull/2167
```

Comment:
I think you were thinking of "superseded-by", but we don't want to use that in this KEP (it is for ones that get replaced by a new KEP). Instead use "see-also" (which takes a list).